app/jobs/regular/translate_categories.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module Jobs
+  class TranslateCategories < ::Jobs::Base
+    cluster_concurrency 1
+    BATCH_SIZE = 50
+
+    def execute(args)
+      return unless SiteSetting.translator_enabled
+      return unless SiteSetting.experimental_category_translation
+
+      locales = SiteSetting.automatic_translation_target_languages.split("|")
+      return if locales.blank?
+
+      cat_id = args[:from_category_id] || Category.order(:id).first&.id
+      last_id = cat_id
```

Comment:
```suggestion
      last_id = nil
```
NIT but a bit easier for me to follow that logic